pkg/apic/consumerinstance.go

```diff
@@ -32,6 +33,20 @@ func (c *ServiceClient) buildConsumerInstanceSpec(serviceBody *ServiceBody, doc
 
 	enableSubscription := c.enableSubscription(serviceBody)
 
+	owningTeam := c.cfg.GetTeamName()
+
+	// If there is an organizationName in the serviceBody, try to find a match in the map of Central teams.
+	// If found, use that as the owningTeam for the service. Otherwise, use the configured default team.
+	if serviceBody.OrganizationName != "" {
+		obj, err := cache.GetCache().Get(TeamMapKey)
+		if err == nil {
+			teamMap := obj.(map[string]string)
+			if _, found := teamMap[serviceBody.OrganizationName]; found {
+				owningTeam = serviceBody.OrganizationName
```

Comment:
Does it buy us anything to log debug if we changed from using default team to use the organizationName?

Reply:
Not sure. Depends on what is "expected" behavior. If we expect that they have everything matched up, then maybe logging that we couldn't find a match would be better.